README.md

```diff
@@ -14,12 +14,10 @@
 </br>
 
 <div align="center">
-  <a href="https://novu.co" target="_blank">
```

Comment:
The changes in this file shouldn't belong to this PR. Looks like you branched off from an old version of the codebase.